frontend_tests/puppeteer_tests/16-settings.js

```diff
@@ -80,6 +80,8 @@ async function test_get_api_key(page) {
     await common.fill_form(page, "#api_key_form", {
         password: test_credentials.default_user.password,
     });
+    // To avoid flake sometimes when running with Firefox.
+    await page.focus(get_api_key_button_selector);
     await page.click(get_api_key_button_selector);
```

Comment:
I think it'd be better to mention what causes the flake and how this fixes it in the commit message.